src/index.js

```diff
@@ -22,35 +38,37 @@ function decode(res) {
   return JSON.parse(jsonPayload);
 }
 
-function backwardCompat(node, attr, key) {
-  return `${node}[data-mystique-key="${key}"],${node}[data-${attr}="${key}"]`;
+function toScript(attr, key, props, id) {
+  return `<script type="application/json" data-${attr}="${key}" data-hypernova-id=${id}>${LEFT}${encode(props)}${RIGHT}</script>`; // eslint-disable-line max-len
```

Comment:
it might be cleaner rather than concatting LEFT and RIGHT, to have an internal "wrap" function - not a huge deal tho obv